azkaban-web-server/src/test/java/azkaban/scheduler/TestQuartzJob.java

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
Other tests are named with xxxTest.java.

Reply:
This is a not test, right? Is SampleQuartzJob a good name?

Reply:
I see. Then the name is fine.